### PR TITLE
Show "Save and Continue" for Progress Notes and Doctors Log

### LINKED ISSUE #8351: Change "Save" to "Save and Continue" for Detailed update and Progress Note
image

### PR DESCRIPTION
**Pull request by shivankacker**
## Proposed Changes

- Fixes #8351

@coronasafe/care-fe-code-reviewers @coronasafe/code-reviewers

## Merge Checklist

- [ ] Add specs that demonstrate bug / test a new feature.
- [ ] Update [product documentation](https://docs.coronasafe.network/coronasafe-care-documentation/architecture/architecture-and-layering-of-care).
- [ ] Ensure that UI text is kept in I18n files.
- [ ] Prep screenshot or demo video for changelog entry, and attach it to issue.
- [ ] Request for Peer Reviews
- [ ] Completion of QA

### CODE REVIEW

**Review by rithviknishad — APPROVED** [review]

File: src/Components/Patient/DailyRounds.tsx [hunk 1]
```diff
@@ -141,7 +141,11 @@ export const DailyRounds = (props: any) => {
   const [showDiscontinuedPrescriptions, setShowDiscontinuedPrescriptions] =
     useState(false);
   const headerText = !id ? "Add Consultation Update" : "Info";
-  const buttonText = !id ? "Save" : "Continue";
+  const buttonText = !id
```
Comment (bodhish) [inline src/Components/Patient/DailyRounds.tsx:144]: Move strings to I18n

**Review by bodhish — CHANGES_REQUESTED** [review]

File: src/Components/Patient/DailyRounds.tsx (hunk 1 above)
Comment (bodhish) [reply src/Components/Patient/DailyRounds.tsx:144]: cc @rithviknishad

**Review by bodhish — APPROVED** [review]
LGTM

**Comment by nihal467** [comment]
LGTM

**Review by khavinshankar — APPROVED** [review]

**Pull request merged**